docker/generic/start_proxy.py

```diff
@@ -328,13 +328,11 @@ def make_argparser():
         if the fields are available. The value must be a primitive field,
         JSON objects and arrays will not be logged.
         ''')
-    parser.add_argument(
-        '--service_control_network_fail_open',
-        default=True,
-        action='store_true',
-        help='''
-        In case of network failures when connecting to Google service control,
-        the requests will be allowed if this flag is on. The default is on.
+    parser.add_argument('--service_control_network_fail_policy',
```

Comment:
This isn't backward compatible, right? If users setting `service_control_network_fail_open=False`. This flag is undefined? 

Actually I just realized for this [pr](https://github.com/cloudendpoints/esp/pull/818), look neither. `service_control_network_fail_open` will always be true even if users setting `service_control_network_fail_open=false` without setting service_control_network_fail_policy, which makes backward uncompatible?

Reply:
For ESPv2,  since it is in beta,  we don't need to worry about backward compatibility.  
Beside you could not use flag --service_control_network_fail_open=False.  as it is tested by users. Please see
[here](https://github.com/cloudendpoints/esp/issues/816) So with old flag, you could not turn if off.

Reply:
SG